### PR TITLE
Multi stat one sample

### LINKED ISSUE #1353: MultiStatistics::terminate throws floating point arithmetic exception
As in the description. The method causes `Signal: SIGFPE (Arithmetic exception)` with f`eenableexcept ( FE_DIVBYZERO | FE_INVALID | FE_OVERFLOW )`. This bug has been discovered thanks to #1352.

### PR DESCRIPTION
**Pull request by copyme**
Fixes #1353

### CODE REVIEW

**Comment by dcoeurjo** [comment]
👌 thx

**Comment by dcoeurjo** [comment]
can you please add a short Changelog entry ?
thx

**Comment by copyme** [comment]
@dcoeurjo done

File: ChangeLog.md [hunk 1]
```diff
@@ -50,6 +50,9 @@
 - *DEC*
   - Adding missing headers in some files of DEC.
     (Roland Denis, [#1349](https://github.com/DGtal-team/DGtal/pull/1349))
+    
+- *Math*
+  - Fix possible division by zero in the MultiStatistics class.
```
Comment (dcoeurjo) [inline ChangeLog.md]:
```suggestion
  - Fix possible division by zero in the MultiStatistics class. (Kacper Pluta, [#1358](https://github.com/DGtal-team/DGtal/pull/1358))
```

Comment (dcoeurjo) [reply ChangeLog.md]: could you please apply my suggestion ? thx

Comment (copyme) [reply ChangeLog.md]: done

**Comment by dcoeurjo** [comment]
👌 thx, merging

**Pull request merged**